pages/mainnet.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <main class="container color-primary">
+    <AppHeader />
+    <div class="content_container">
+      <div class="center">
+        <h1 class="heading font-sans">Mainnet Launching Soon</h1>
+        <p class="heading_caption color-secondary font-serif">
+          Join 50+ Web3 projects and 1000s of devs building scalable dApps using
+          Arcana’s Auth, Store and Access
+        </p>
+      </div>
+      <div class="center">
+        <div v-if="loading" class="font-serif">Please Wait...</div>
+        <div v-else class="center">
+          <form v-if="!success" class="email_input_container font-serif">
+            <input
+              v-model="email"
+              type="email"
+              placeholder="Your email address"
+              class="email_input_field"
+            />
+            <button
+              class="email_input_button cursor-pointer"
+              @click.prevent="signup"
+            >
+              Notify Me
+            </button>
+          </form>
+          <div v-else class="email_success_message font-serif">
+            <p>AWESOME! YOU’RE ON THE LIST!</p>
+          </div>
+          <v-text class="subscription-error-message">
+            {{ message }}
+          </v-text>
+        </div>
+        <p class="email_input_caption color-secondary font-serif">
+          Provide your email address to stay updated, get giveaways, NFT drops
+          and more. We won’t spam, promise!
+        </p>
+      </div>
+    </div>
+    <MainnetFooter />
+  </main>
+</template>
+
+<script>
+import { groupTypes, subscribe } from '~/services/mailchimp'
+
+export default {
+  layout: 'default',
+  data() {
+    return {
+      email: '',
+      success: false,
+      message: '',
+      loading: false,
+    }
+  },
+  head() {
+    return {
+      title: 'Mainnet - Arcana Network',
+    }
+  },
+  methods: {
+    async signup() {
+      this.loading = true
+      if (this.email.trim()) {
+        try {
+          await subscribe({
+            email: this.email.trim(),
+            groups: [groupTypes.MAINNET],
+          })
+          this.success = true
+        } catch (e) {
+          this.success = false
+          if (/subscribed/.test(e)) {
+            this.message = 'Already Subscribed'
+          } else if (/0 - /.test(e)) {
+            this.message = 'Invalid email'
+          } else {
+            this.message = 'Something went wrong'
+          }
+        }
+      } else {
+        this.success = false
+        this.message = 'Enter your email address'
+      }
+      this.loading = false
+    },
+  },
+}
+</script>
+
+<style lang="postcss" scoped>
+@import url('../components/lib/media-query-helper.css');
+
+.container {
+  background: linear-gradient(
+      344.06deg,
+      rgba(0, 0, 0, 0) -94.53%,
+      rgba(0, 0, 0, 0.62) 79.99%
+    ),
+    url('~assets/images/mainnet-bg.svg') no-repeat;
```

Comment:
I guess these images can be png images since they're proper images and not vectors

cc: @ajithranka

Reply:
SVG loads quickly, IMO. @ajithranka your feedback?